test/compress/issue-203.js

```diff
@@ -49,8 +49,8 @@ compress_new_function_with_destruct_arrows: {
         new Function("[[aa]], [{bb}]", 'return aa;');
     }
     expect: {
-        Function("n,[a]", "return n");
-        Function("b,{bb:n}", "return b");
-        Function("[[b]],[{bb:n}]", "return b");
+        Function("n,[r]", "return n");
+        Function("n,{bb:b}", "return n");
+        Function("[[n]],[{bb:b}]", "return n");
```

Comment:
I don't understand the interaction of mangle with `unsafe_Func`, but I'll let it ride.

Using `unsafe_arrows` instead of `arrows` for this test still results in different `expect` output from the original:
```js
---OUTPUT---
Function("n,[r]","return n");Function("b,{bb:n}","return b");Function("[[b]],[{bb:n}]","return b");
---EXPECTED---
Function("n,[a]","return n");Function("b,{bb:n}","return b");Function("[[b]],[{bb:n}]","return b");
```
Should I add more `r`'s to the test case or leave test as is with `arrows`?